packages/flutter/lib/src/services/system_channels.dart

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'platform_channel.dart';
+import 'message_codecs.dart';
+
+/// A JSON [PlatformMethodChannel] for navigation.
+const PlatformMethodChannel flutterNavigationChannel = const PlatformMethodChannel(
```

Comment:
This would be a bit cleaner if there was a class that these channels all belonged to as static members.  Perhaps:

`SystemChannels.navigation`
`SystemChannels.platform`

etc